packages/devtools_app/lib/src/screens/memory/panes/diff/diff_pane.dart

```diff
@@ -30,58 +30,39 @@ class _DiffPaneState extends State<DiffPane> {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder<int>(
+    final Widget listContent = ValueListenableBuilder<int>(
```

Comment:
nit: rename to snapshotView since this is not actually content in a list - it's content for a selected snapshot item in a list.